src/qudi/gui/time_series/time_series_gui.py

```diff
@@ -299,6 +304,20 @@ def on_deactivate(self):
 
         self._mw.close()
 
+    @_view_settings.representer
+    def __repr_view_settings(self, value):
+
+        view_settings = {}
+
+        vis_settings = self.visible_settings
+        curr_ch = self._mw.curr_value_comboBox.currentText()
+        curr_ch_idx = self._mw.curr_value_comboBox.findText(curr_ch)
+
+        view_settings['current_channel_idx'] = curr_ch_idx
```

Comment:
I think it would make more sense, to serialize channel string here instead of the index since upon change of the channels in the hardware, the gui would display a different channel on reload of the module while I think it should go to None in this case.

Reply:
Should be resolved with 77cd1120dd4a7a65f8255fbb24b1e0e59a087a80